### PR TITLE
A small add to the script

### PR DESCRIPTION
**Pull request by mayabrandi**
I also set the "Include reads" field automatically based on QC. Would like to have this merged asap

### CODE REVIEW

File: scripts/undemultiplexed_index.py [hunk 1]
```diff
@@ -171,8 +173,10 @@ def _QC(self, target_file, sample_info):
         QC2 = (Q30 >= self.QC_thresholds['%Q30'])
         QC3 = (self.read_pairs >= self.QC_thresholds['nr_read'])
         if QC1 and QC2 and QC3:
```
Comment (Galithil) [inline scripts/undemultiplexed_index.py]: could you add

``` and 'Include reads' in taget_file.udf ```

? This should ensure that the script does not fail if the udf field is not set

Comment (mayabrandi) [inline scripts/undemultiplexed_index.py]: The field is not set. It is being set here. So it wont fail @Galithil

**Comment by Galithil** [comment]
fine with it, ewe might have to meet with the lab to try to normalize fields usage between miseq and hiseq

**Comment by ewels** [comment]
ewe

**Pull request merged**